app/subscriber/src/features/utils/showTranscription.ts

```diff
@@ -0,0 +1,9 @@
+import { ContentTypeName, IContentModel } from 'tno-core';
```

Comment:
Separate function for viewing content as other function was for previews not full length